### PR TITLE
SRCH-15 fix suggester bug

### PR DESCRIPTION
**Pull request by MothOnMars**
- send query errors to NewRelic
- log full query JSON for failed queries

### CODE REVIEW

File: app/classes/document_search.rb [hunk 1]
```diff
@@ -19,9 +19,11 @@ def search
...
     end
     i14y_search_results
-  rescue StandardError => e
-    Rails.logger.error "Problem in DocumentSearch#search(): #{e}
-    #{e.backtrace}"
+  rescue StandardError => error
+    Rails.logger.error "Problem in DocumentSearch#search(): #{error}
+    Query: #{doc_query.body.to_json}
+    Backtrace: #{error.backtrace}"
+    NewRelic::Agent.notice_error(error, options: { custom_params: { indices: indices }})
```
Comment (peggles2) [inline app/classes/document_search.rb]: Can we indent Query and Backtrace to line up with "Problem in DocumentSearch..." That way we know thats all part of the logger.error.

Comment (MothOnMars) [reply app/classes/document_search.rb]: Good idea - will do

**Comment by MothOnMars** [comment]
@peggles2 , do you approve of my last `squashme` commit? 
https://github.com/GSA/i14y/pull/52/commits/73d46f104b7a0cea42ac79d0dcca3f7043db917f

**Comment by peggles2** [comment]
LGTM 👍

**Pull request merged**